"Generating a client" instructions outdated?

I followed the [instructions in the `README` for generating a client](https://github.com/awslabs/smithy-typescript#generating-a-client); as instructed I have a `build.gradle.kts`, a `smithy-build.json`, and a `model` folder with the model file in it. 

I get the following error when I run `gradle build`:

<details>
<summary>Failed build output</summary>

```sh
foo-client % gradle build
Starting a Gradle Daemon (subsequent builds will be faster)
(assuming Smithy CLI version 0.9.7)
> Task :smithyBuildJar FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':smithyBuildJar'.
> Could not resolve all files for configuration ':smithyCli'.
   > Cannot resolve external dependency software.amazon.smithy:smithy-typescript-codegen:0.1.0 because no repositories are defined.
     Required by:
         project :
   > Cannot resolve external dependency software.amazon.smithy:smithy-cli:0.9.7 because no repositories are defined.
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
1 actionable task: 1 executed
```

</details>

Expected output:

<details>
<summary>Successful build output</summary>

```sh
foo-client-3 % gradle build                          
(detected Smithy CLI version 1.12.0)
Building Smithy model sources: [/.../foo-client/model]
Loading Smithy configs: [/.../foo-client/smithy-build.json]
Validation result: 0 ERROR(s), 0 DANGER(s), 0 WARNING(s), 0 NOTE(s)
Validated 370 shapes in model
Completed projection source (370 shapes): /.../foo-client/build/smithyprojections/foo-client/source
Smithy built 1 projection(s), 4 plugin(s), and 20 artifacts

> Task :smithyBuildJar
Running smithy build
Validating Smithy model sources: []
Validation result: 0 ERROR(s), 0 DANGER(s), 0 WARNING(s), 0 NOTE(s)
Validated 370 shapes in model
Smithy validation complete

> Task :smithyValidate
Running smithy validate

BUILD SUCCESSFUL in 1s
4 actionable tasks: 4 executed
```

</details>

The build is successful if I update the `build.gradle.kts` file:

<details>
<summary>diff</summary>

```diff
--- a/foo-client/build.gradle.kts
+++ b/foo-client/build.gradle.kts
@@ -1,7 +1,18 @@
 plugins {
-    id("software.amazon.smithy").version("0.4.3")
+    id("software.amazon.smithy").version("0.5.3")
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
 }
 
 dependencies {
-    implementation("software.amazon.smithy:smithy-typescript-codegen:0.1.0")
+    implementation("software.amazon.smithy.typescript:smithy-typescript-codegen:0.6.0")
+
+    implementation("software.amazon.smithy:smithy-model:1.12.0")
+
+    // These are just examples of dependencies. This model has a dependency on
+    // a "common" model package and uses the external AWS traits.
+    implementation("software.amazon.smithy:smithy-aws-traits:1.12.0")
 }
```

</details>

I'm not sure what else might need to be changed, but this seems like a start. 

Would you be open to a PR that updates the README?

--- a/README.md
+++ b/README.md
@@ -17,13 +17,19 @@ are as follows:
 2. Create a `build.gradle.kts` file with the following contents:
 
    ```kotlin
-   plugins {
-       id("software.amazon.smithy").version("0.4.3")
-   }
+    plugins {
+        id("software.amazon.smithy").version("0.6.0")
+    }
 
-   dependencies {
-       implementation("software.amazon.smithy:smithy-typescript-codegen:0.1.0")
-   }
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        implementation("software.amazon.smithy:smithy-model:[1.16.0, 2.0[")
+        implementation("software.amazon.smithy:smithy-typescript-codegen:0.8.0")
+    }
    ```
 
 3. Create a `smithy-build.json` file with the following contents,
@@ -32,29 +38,56 @@ are as follows:
    create.:
 
    ```json
-   {
-       "version": "1.0",
-       "plugins": {
-           "typescript-codegen": {
-               "service": "example.weather#Weather",
-               "package": "weather",
-               "packageVersion": "0.0.1",
-               "packageJson": {
-                   "license": "Apache-2.0"
-               }
-           }
-       }
-   }
+    {
+        "version": "1.0",
+        "plugins": {
+            "typescript-codegen": {
+                "service": "smithy.example#ExampleService",
+                "targetNamespace": "SmithyExample",
+                "package": "smithyexample",
+                "packageVersion": "0.0.1",
+                "packageJson": {
+                    "license": "Apache-2.0"
+                }
+            }
+        }
+    }
 
    ```
 
 4. Create a directory named `model`. This is where all of your Smithy models
    will go.
 
-5. Run `gradle build` (alternatively, you can use a
+5. Create a file in the `model` directory named `main.smithy` with the following contents:
+
+    ```
+    namespace smithy.example
+
+    service ExampleService {
+        version: "2022-01-01",
+        operations: [EchoOperation]
+    }
+
+    operation Echo {
+        input: EchoInput,
+        output: EchoOutput,
+    }
+
+    @input
+    structure EchoInput {
+        message: String,
+    }
+
+    @output
+    structure EchoOutput {
+        message: String
+    }
+    ```
+
+6. Run `gradle build` (alternatively, you can use a
    [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html)).
 
-6. The generated client can be found in `build/smithyprojections/foo-client/source/typescript-codegen`.
+7. The generated client can be found in `build/smithyprojections/foo-client/source/typescript-codegen`.
 
 See [the Smithy documentation](https://awslabs.github.io/smithy/guides/building-models/gradle-plugin.html)
 for more information on build Smithy projects with Gradle.

